src/app/modules/users/groups/groups.component.scss

```diff
@@ -0,0 +1,26 @@
+@import '../../../../scss/constants/colors';
```

Comment:
U can use short name for import. Please see https://scotch.io/tutorials/angular-shortcut-to-importing-styles-files-in-components

Reply:
Fixed